website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -190,6 +190,10 @@ the [`sidecar_service`](/consul/docs/connect/registration/sidecar-service) block
 - `envoy_stats_flush_interval` - Configures Envoy's
   [`stats_flush_interval`](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-stats-flush-interval).
 
+- `envoy_telemetry_collector_bind_socket_dir` - Specifies the directory where Envoy creates a unix socket. 
```

Comment:
This should have been the right place to have the docs, so the docs were moved from `website/content/commands/connect/envoy.mdx` to this file.